### ISSUE TITLE
json files starting with square brackets in XConfig

### ISSUE THREAD

**Issue by astrodave85** [eyecan-ai/pipelime-python#98]
* pipelime version: 1.2.0
* Python version: 3.9
* Operating System: Win

### Description

I'm using the XConfig Class for useful imports from files to abstract away whether a file is a yaml or a json, i.e. like:
```
XConfig.from_file(whatever_file).to_dict()
```
But I've recently encountered json files formed by sequences (very rare indeed) that are failing. Do you think that they may be planned to be integrated in the future?

### What I Did

```
data_dict = {"foo": 1}
data_array = [1, 2, 3, 4, 5]
from collections import Mapping, Sequence
from pipelime.choixe import XConfig

# a mapping succeeds
XConfig(data_dict)
# a sequence fails
XConfig(data_array)

```

**Comment by nfioraio-ec** [comment]
Yeah, unfortunately XConfig currently wants a dict as root. However, if you just need to load a simple json/yaml, pyyaml is enough since any json _is_ a valid yaml file:

```python
import yaml

with open("data.json") as f:
    data = yaml.safe_load(f)  # it works with json file!
```

**Comment by astrodave85** [comment]
Thanks for the suggestion!

**Comment by nfioraio-ec** [comment]
See this merged PR #111 --> now you can load pretty much anything using XConfig, see the docs https://pipelime-python.readthedocs.io/en/latest/choixe/xconfig.html?h=xconfig#unsafe-processing-variant
